.travis.yml

```diff
@@ -17,20 +17,22 @@ env:
   global:
     - THIS_REPO=checker-framework-inference
     - OS=ubuntu
-    - JDKVER=jdk8
 
   matrix:
-    - GROUP=cfi-tests
-    - GROUP=downstream-ontology
-    - GROUP=downstream-units
+    - GROUP=cfi-tests JDKVER=jdk8
```

Comment:
Reminder: we should update `.travis.yml` when pulling typetools/mater into opprop, by having groups with JDK8 and groups with JDK11.